### DISCUSSION TITLE
Getting "Call to undefined method Imdhemy\\Purchases\\Subscription::verify()"

### DISCUSSION THREAD

**Discussion by jamesbhatta** [imdhemy/laravel-in-app-purchases#348] [General]
Hi there. I had a fresh install of the pakage and configured app store credentials.Calling `` $receiptResponse = Subscription::appStore()->receiptData($receipt)->verify();`` throws an error ``Call to undefined method Imdhemy\\Purchases\\Subscription::verify()``. Do I need to change ``verify()`` to ``verifyReceipt()`` for renewal subscriptions?

**Comment by imdhemy** [comment]
Hi @jamesbhatta 
Sorry for that, It's a typo in the documentation and I will fix it. 

You can use either `toStd()` which should work for Google play and App Store, or you can use `verifyReceipt()` for App Store.

**Comment by jamesbhatta** [comment]
Hey @imdhemy Thank you very much.
